getMessages.js

```diff
@@ -0,0 +1,70 @@
+const jsonfile = require('jsonfile');
+const preq = require('preq');
+
+const API_BASE_URL = `https://test.wikipedia.org/w/api.php`;
```

Comment:
Would be better to use metawiki or mediawikiwiki as vandalism is less likely there. Or use the default message instead of the local one.

Reply:
makes sense, done